tests/providers/aws/aws_provider_test.py

```diff
@@ -249,7 +250,10 @@ def test_aws_provider_default(self):
         arguments = Namespace()
         arguments.mfa = False
         arguments.scan_unused_services = True
```

Comment:
You can remove all this references in this file because namespace is no longer needed.

Reply:
Done!